packages/tldraw/src/test/bindings.test.tsx

```diff
@@ -246,9 +246,6 @@ test('copying the from shape on its own does trigger isolation operations', () =
 		  "onBeforeDelete",
 		  "onAfterDelete",
 		  "onOperationComplete",
-		  "onBeforeCreate",
```

Comment:
@ds300 Saw that you wrote these tests. Are there any observable issues if these don't get triggered?

Reply:
Reworked this.